.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ repos:
     hooks:
       #- id: check-yaml
       #  args: [--allow-multiple-documents]
-      # - id: end-of-file-fixer
+      - id: end-of-file-fixer
```

Comment:
Why was this disabled in the first place?

Reply:
@aledbf do you know the reason? (context: https://github.com/gitpod-io/gitpod/commit/f65deb95b14fdf6658b14445b3252026120818c9)

Reply:
It was related to the number of changes at the time of introduction of `pre-commit`